src/ProfilesList.json

```
@@ -251,9 +251,10 @@
   "shreyasdeshpande.json",
   "YashSalunke12.json",
   "pratap360.json",
+  "gaurovgiri.json"
```

```suggestion
  "gaurovgiri.json",
```
Add comma `,` after your filename @gaurovgiri